README.md

```diff
@@ -1,5 +1,5 @@
 # Python Client
-Programmatically 💻 interact with Tamr Unify using Python 🐍
+Programmatically 💻 interact with Tamr using Python 🐍
 
 [![Version](https://img.shields.io/pypi/v/tamr-unify-client.svg?style=flat-square)](https://pypi.org/project/tamr-unify-client/)
```

Comment:
Note: references to `tamr-unify-client` have to stay this way until we publish as `tamr-client` to PyPI